Sources/Sentry/Public/SentryOptions.h

```diff
@@ -766,6 +766,29 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, readonly) SentryExperimentalOptions *experimental;
 
+#if TARGET_OS_IOS && SENTRY_HAS_UIKIT
+
+/**
+ * Block used to configure the user feedback widget, form, behaviors and submission data.
+ */
+API_AVAILABLE(ios(13.0))
+typedef void (^SentryUserFeedbackConfigurationBlock)(
+    SentryUserFeedbackConfiguration *_Nonnull configuration);
+
+/**
+ * A block that can be defined that receives a user feedback configuration object to modify.
+ * @warning This is an experimental feature and may still have bugs.
+ * @note This is unrelated to @c SentrySDK.captureUserFeedback which is the old method of submitting
+ * user feedback you've already gathered via your own UI
+ * (see https://docs.sentry.io/platforms/apple/user-feedback/#user-feedback-api). The new method
+ * uses either this block to configure and a widget and UI form to gather feedback, or directly
+ * submits calling the new method @c SentrySDK.captureFeedback (see
+ * https://docs.sentry.io/platforms/apple/user-feedback/configuration/).
+ */
+@property (nonatomic, copy, nullable)
```

Comment:
`l`: Is there any value in not configuring this in the root options, but instead moving it to something like `options.userFeedback.configure { ... }`, implementing the nested options in Swift instead? Similar to the experimental options

Reply:
From the perspective of SentryOptions, this is no different, since there is only one point of entry to the user feedback configuration, which is SentryOptions.configureUserFeedback, and everything else is contained in there.